src/fake-timers-src.js

```diff
@@ -505,6 +505,8 @@ function withGlobal(_global) {
             return formatter;
         };
 
+        ClockIntl.RelativeTimeFormat = NativeIntl.RelativeTimeFormat;
```

Comment:
Are there others? Could we perhaps loop over the props of `NativeIntl`?

Reply:
@SimenB yeah there are other methods, and we could loop over them and assign them to `ClockIntl` with something like this.

```js
Object.getOwnPropertyNames(NativeIntl)
    .filter((item) => typeof NativeIntl[item] === "function")
    .forEach((method) => (ClockIntl[method] = NativeIntl[method]));
```

We still would need to keep the new implementation of `DateTimeFormat` due to this LOC
https://github.com/sebastinez/fake-timers/blob/f4469bf39f933b148e400170ae7d698c41079908/src/fake-timers-src.js#L505
Which allows to pass `undefined` to `Intl.DateTimeFormat` here
https://github.com/sebastinez/fake-timers/blob/f4469bf39f933b148e400170ae7d698c41079908/test/fake-timers-test.js#L5193

Reply:
I was not merging due to this seemingly being unresolved .. ?

Reply:
@simenb any objections to his proposal?

Reply:
not at all 🙂

Reply:
PS. The line of code you are referring to is just implementing the Ecmascript spec 
https://tc39.es/ecma402/#sec-datetime-format-functions